src/js/chartAuthoring.js

```diff
@@ -66,6 +68,12 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                                 },
                                 styles: {
```

Comment:
Line 66-71 are common options used by both fluid.inline components. To reduce the repetition, we can defined a top option block such as "inlineEditCommonOptions" that contains these lines then use distributeOptions to distribute them into all "fluid.inlineEdit".